vajram/src/main/java/com/flipkart/krystal/vajram/inputs/ResolveInputsOf.java

```diff
@@ -4,8 +4,8 @@
 import java.lang.annotation.RetentionPolicy;
 
 @Retention(RetentionPolicy.RUNTIME)
-public @interface Resolve {
-  String value();
+public @interface ResolveInputsOf {
```

Comment:
Will prefer name - `ResolveDep` with fields `depName` and `depInputs`. @RamAnvesh fyi

Reply:
@prateek-kr-fk addressed comments

Reply:
Yes ResolveDep is better